src/parser.js

```diff
@@ -440,25 +433,6 @@
     return selector.length === 0;
   }
 
-  /**
-   * @private
-   * to support IE8 missing getElementById on SVGdocument
-   */
-  function elementById(doc, id) {
-    var el;
-    doc.getElementById && (el = doc.getElementById(id));
-    if (el) {
-      return el;
-    }
-    var node, i, len, nodelist = doc.getElementsByTagName('*');
-    for (i = 0, len = nodelist.length; i < len; i++) {
-      node = nodelist[i];
-      if (id === node.getAttribute('id')) {
-        return node;
-      }
-    }
-  }
```

Comment:
it was meant for old version of ie. let's see if it works fine without in node and ie10+ in case i will add it back